feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -1,153 +1,185 @@
-# TE-18.1: gRIBI MPLS in UDP Encapsulation
+# TE-18.1: gRIBI MPLS-in-UDP Encapsulation
 
-Create AFT entries using gRIBI to match on next hop group in a
-network-instance and encapsulate the matching packets in MPLS in UDP with outer header as IPv6 Header.
+## Summary
 
-The MPLS in UDP encapsulation is expected to follow
-[rfc7510](https://datatracker.ietf.org/doc/html/rfc7510#section-3),
-but relaxing the requirement for a well-known destination UDP port.  gRIBI is
-expected to be able to set the destination UDP port.
+Test MPLS-in-UDP encapsulation using gRIBI to create AFT entries that
+match IPv6 traffic and encapsulate it with MPLS labels inside UDP
+packets with IPv6 outer headers. The test validates that the DUT
+correctly implements RFC 7510 MPLS-in-UDP encapsulation with
+configurable UDP destination ports.
 
 ## Topology
 
-* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+ATE port-1 \<——\> port-1 DUT
 
-## Test setup
+DUT port-2 \<——\> port-2 ATE
 
-TODO: Complete test environment setup steps
+- [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
```

Comment:
do we need to include 3 ports to ensure load balancing on egress occurs? Or is that covered in TE-18.3?